apps/elewa-website/src/app/app.routing.ts

```diff
@@ -10,8 +10,13 @@ export const ELEWA_WEBSITE_ROUTES: Route[] = [
   { path: 'home', loadChildren: () => import('@elewa-website/features/pages/home').then(m => m.FeaturesPagesHomeModule) },
 
   { path: 'about', loadChildren: () => import('@elewa-website/features/pages/about').then(m => m.FeaturesPagesAboutModule) },
+
+  { path: 'cards', loadChildren: () => import('@elewa-website/elements/cards').then(m => m.CardsModule) },
+
```

Comment:
No need for the route as this is not a page

Reply:
For me to merge this, Yes as it's redundant.

Reply:
okay solved